apps/src/applab/applabObjectFitImages.js

```diff
@@ -0,0 +1,13 @@
+import objectFitImages from 'object-fit-images';
+
+/**
+ * Wrapper to objectFitImages() to avoid using it on phantomjs
+ * See https://github.com/bfred-it/object-fit-images/issues/11
+ */
+
+export function applabObjectFitImages(imgs, opts) {
+  if (/PhantomJS/.test(window.navigator.userAgent)) {
```

Comment:
:+1: Thanks for doing this instead of IN_UNIT_TEST - that means if we switch to headless chrome we'll actually run this polyfill in tests.